website/components/blog/Recommendation.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import PostPreview, { Post } from "./PostPreview"
+
+export interface Props {
+  title: string
+  posts: Array<Post>
+}
+
+const Recommendation: React.FC<Props> = ({ title, posts }) => {
```

Comment:
When we have a component that just return JSX, we can skip curly brackets `{}` and `return` statement like this:

```
const SomeComponent = () => (<div>asdasd</div>)
```

Reply:
But this is minor stylistic suggestion

Reply:
well, not only in JSX, just ES6 AF implicit return. Yet often ends in some props destructuring needed/variable for code readability added and component have to be restyled.